packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -2,52 +2,98 @@
   * `access_token` &mdash; this token is restricted to give access only to what the end user authorized for your application
   * `token_type=Bearer` &mdash; this identifies the authentication method used
   * `expires_in` &mdash; the expiration time for the access token (in seconds)
   * `scope` &mdash; the scopes that are contained in the access token
   * `id_token` &mdash; the ID token contains the specific info (in JWT format) about the authentication and the end user that is needed by the application
   * Optional. `refresh_token` &mdash; this token is returned if the `offline_access` scope is granted. This token can be exchanged for a new access token if the previous access token has expired
-* The access token is then used by your application to retrieve the protected resources and information requested at the Resource Server for the end user
+* The access token is then used by your application to retrieve the protected resources and information requested at the Resource Server for the end user.
 
-The authorization code flow depends on Okta and your SaaS back-end systems being able to securely communicate the Client Secret.
+The Authorization Code flow depends on Okta and your SaaS back-end systems being able to securely communicate the Client Secret.
 
-Note that your application needs to verify user identity through a user store, otherwise your app could be spoofed by fake applications in the Android or Apple app stores, or an end user could view and possibly modify your code. For this reason, native, mobile, and single page application (SPA) integrations are not acceptable for OIDC app integrations in the OIN. Your application code must be set up in a flow so that your client application talks to your SaaS back-end which then communicates with Okta.
+Note that your application needs to verify user identity through a user store. Otherwise, your app could be spoofed by fake applications in the Android or Apple app stores, or an end user could view and possibly modify your code. For this reason, native and mobile app integrations are not acceptable for OIDC app integrations in the OIN. You must set up your application to use an authentication flow so that your client application talks to your SaaS back-end, which then communicates with Okta.
 
 To support the potentially large numbers of Okta orgs accessing it through the OIN, an OIDC integration can't use a custom authorization server, including the `default` server. You can only use the [Org Authorization Server](https://developer.okta.com/docs/concepts/auth-servers/#available-authorization-server-types).
 
-Another general outline of the authorization code flow can be found in our Okta developer guide: [Implement the Authorization Code Flow](/docs/guides/implement-auth-code/overview/)
+Another general outline of the Authorization Code flow is in our Okta developer guide: [Implement the Authorization Code Flow](/docs/guides/implement-auth-code/overview/)
```

Comment:
```suggestion
Another general outline of the Authorization Code flow is in our Okta developer guide: [Implement the Authorization Code flow](/docs/guides/implement-auth-code/overview/)
```

Reply:
Need to update the guide titles to match